### PR TITLE
UAC Ambition - (New atmospherics ship)

### PR DESCRIPTION
**Pull request by Cu1r**
## About the PR
The PR adds the UAC Ambition, a gigantic new ship for gas acquisition and processing.

## Why / Balance
Their are few current options for dedicated atmospherics gameplay (and for good reason). Gas refinement requires a large space to experiment and play with, as well as the ability to procure and store crucial resources. As a result, none of the current ships on the market have a size currently to my liking, nor storage space for the amount of gas necessary to make atmospherics reliable and desirable as gameplay. I also wanted to include PROPER fire safety protocols, air alarm use, and a mix chamber. For sure, I don't expect atmos to be anything near what it once was anytime soon, but I want to give players the best shot they can shoot when it comes to trying to turn a profit, within reason. This happens to be a pretty cheap ship for it's size at 132 thousand spesos. It is not a luxury yacht, so I hope to see people trying it out in the future.

## How to test
I'd like some feedback on the manueverabilty of the ship (is it too fast maybe?) since I want it to be able to pull up to wrecks and grab canisters relatively quickly and with no issue. Some of the larger ships are very slow and frustrating (probably because they do expeds) and I feel that would really cripple this ship if it was too slow.
This is not supposed to be a salvage ship by trade, but atmospherics gameplay requires salvaging canisters as a bare minimum, so whatever that minimum is suppose to be, I'd like some feedback on that as well. This is also not an expeditions ship. I want this ship to be available (in the Frontier shipyard) to as many people as possible for the sole purpose of what it was designed to do. This big duck is not landing on any planets. That would be very silly.
I'm not so great with minute details after a certain point. I can think up a few items, but if something crucial is missing, it's hard to tel. There is plenty of room left to add details and I'd really appreciate it if some people could give some suggestions for the scattered little empty spots left in the ship.

## Breaking changes
This ship extends to a length of 60 meters, longer than the 48 maximum recommended by the size chart. For this reason I made sure to place double docks at key points on the ship to give it the best chance of staying out of the way, despite its size. I don't think it's anywhere near the maximum tile count of 2304 (which I believe is the non-negotiable part), so I hope that the size is utilized effectively to the standard that allows it to pass inspection. It is a bit bigger than a Praeda, and probably a Gourd as well, but is still much smaller in area than a Crescent.

## Media
uac ambition
iff
ambition map

- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

**Changelog**
:cl:
- add: The UAC Ambition, an Atmosian logistical powerhouse, makes an appearance in the civilian dockyard after being declassified following its seizure by NanoTrasen black ops.

### CODE REVIEW

**Comment by dvir001** [comment]
Its been some time now since I seen a ship that didn't try to add a full medical and a full kitchen when it had no need for it.

Based.

**Comment by dvir001** [comment]
Honestly, I think this ship should be able to go on expo for gas collections, can you try making it an expo ship and see if its size isn't breaking the fabric of time and space when going on planets?

**Comment by muffe07** [comment]
Im just a bit sad it doesn't have a TEG, considering its size and purpose.

**Comment by dvir001** [comment]
> Im just a bit sad it doesn't have a TEG, considering its size and purpose.

Lag machine and a bit not practical for frontier.

**Comment by muffe07** [comment]
Obviously the AME is just way better. But the option would be nice to have the option. 
Didn't know it was laggy though. That too bad.

**Comment by dvir001** [comment]
> Obviously the AME is just way better. But the option would be nice to have the option. Didn't know it was laggy though. That too bad.

Its not bad when you have one in a server but the moment you have alot of them you notice.

**Comment by dvir001** [comment]
> > Honestly, I think this ship should be able to go on expo for gas collections, can you try making it an expo ship and see if its size isn't breaking the fabric of time and space when going on planets?
> 
> If you believe that wouldn't be too gamey, I'd be happy to add and test that functionality.

Go for it, I think its fine.

File: Resources/Prototypes/_NF/Shipyard/ambition.yml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+  shuttlePath: /Maps/_NF/Shuttles/ambition.yml
+
+- type: gameMap
+  id: Ambition
+  mapName: 'UAC Ambition'
+  mapPath: /Maps/_NF/Shuttles/ambition.yml
+  minPlayers: 0
+  stations:
+    Ambition:
+      stationProto: StandardFrontierVessel
```
Comment (Cheackraze) [inline Resources/Prototypes/_NF/Shipyard/ambition.yml:21]: this needs to be StandardFrontierExpeditionVessel in order to allow expeditions for the ship

**Comment by Cheackraze** [comment]
I would say the design of the ship overall is great, but the only thing preventing a full on 10/10 is it needs jut a bit more fine decal work. Specifically, around things like your connector ports, lockers, and machines. Use color coded dotted line box decal around connector ports for example, or even full blown color coded diagonal hazard line decals on them. Emphasize the places on the ship that have those permanent fixtures.  Can storage as well could use some decals. Atmos and salvage locker rooms could use departmental color coded accent decals.

Its actually a fantastic ship design and pretty well done, i just think it could look even better.

**Comment by dvir001** [comment]
image

What server is this?

**Comment by Cheackraze** [comment]
> Picture is a bit outdated, I can get you a new one. Department colors I can do, and I'll see what else works as well.

yea some of the rooms just look a little too much like just blank liminal spaces with some objects thrown in. Its the little touches like highlighting connector ports, hazard lines sectioning off places where machines are supposed to sit, and designating storage spots for like cargo space. just those little fine details that really make a ship pop

**Comment by Cheackraze** [comment]
i will say im in love with this ship design, it reminds me of one my first maps quite a bit

**Comment by Cheackraze** [comment]
latest dotnet minor update causing tests to fail, sorry about that

**Comment by Cheackraze** [comment]
> > i will say im in love with this ship design, it reminds me of one my first maps quite a bit
> 
> I can honestly say that Kettle was my biggest inspiration a long time ago when it first came out. Still one of my favorites to this day. Always reminded me that proper placement of details was important. Thank you very much.
> 
> I have an updated image for you now. I hope everything seems in order. Feel free to continue critiquing. Sometimes it can take another person being like 'oh, that might fit right there' or something to make it click, but I think I've done a good job with the actual issues so far:
> 
> ambition

now thats looking very nice, basically all those empty liminal spaces feel so much more alive and useful

**Comment by Cheackraze** [comment]
> Still failing currently, but I suspect you'll be sorting that out at some point for good and I'll fix my stuff whenever it is. This can hold off until then. Of course, you'll have priorities anyway.

Is a dotnet thing. You can try pulling master again, we have it pinned to an older version for now to try to get them to pass. Ultimately we either wait for dotnet to change or what for our libraries to update, whichever happens

**Comment by dvir001** [comment]
> Ok that's better. Now we just need that infiltrator PR.

You can pull again,
Also mind adding a guidebook for your ship?

image

**Review by Cheackraze — APPROVED** [review]

**Comment by BluCottonCandy** [comment]
I was wondering, is it meant to be missing a cargo pad for completed expeds?
As of right now, you get no reward for completing expeds.

**Comment by dvir001** [comment]
> I was wondering, is it meant to be missing a cargo pad for completed expeds? As of right now, you get no reward for completing expeds.

Should be mapped, but right now the long term goal is to give it "Atmost" expo  and remove it later.

**Comment by BluCottonCandy** [comment]
> > I was wondering, is it meant to be missing a cargo pad for completed expeds? As of right now, you get no reward for completing expeds.
> 
> Should be mapped, but right now the long term goal is to give it "Atmost" expo and remove it later.

I looked everywhere on the ship after doing a few expeditions, but found no cash reward for completing the expedition. 
Unless you mean it's been added since then.

**Comment by BluCottonCandy** [comment]
> > > I was wondering, is it meant to be missing a cargo pad for completed expeds? As of right now, you get no reward for completing expeds.
> > 
> > 
> > Should be mapped, but right now the long term goal is to give it "Atmost" expo and remove it later.
> 
> It's not mapped, but its a bummer if people expect rewards and don't get them, for as little time as this is planned to go to normal expeds. I can quickly map it though, even if it will probably be removed later.

It's to bad this ship is leaving not al expeds for Atmos only expeds. 
Though until the special exped is implemented I personally feel it would be good to add it for now at least.

**Comment by azigler** [comment]
Really love this ship, it's my new favorite. One small detail: the connector port for the condenser is unusable unless you deconstruct the corner railing on the tile currently.

**Comment by Flam3masterr** [comment]
Hello! 
you installed non-passive ventilation outside
03-06-2024_WEB_V_5973

**Comment by dvir001** [comment]
> Hello! you installed non-passive ventilation outside. Why? 03-06-2024_WEB_V_5973

How else are you going to suck atmosphere in?

**Comment by Flam3masterr** [comment]
> > Hello! you installed non-passive ventilation outside. Why? 03-06-2024_WEB_V_5973
> 
> How else are you going to suck atmosphere in?

Maybe we need scrubbers with settings?

**Pull request merged**